README.md

```diff
@@ -61,7 +61,15 @@ Documentation of the LSST Science Pipelines is at https://pipelines.lsst.io
 4. Alternatively, import raw and calibration data into the data repository, for example:
 
         $ ingestImagesDecam.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
-        $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 999
+        $ ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ --calibType defect /path/to/calib/*fits --validity 0
```

Comment:
FYI, the argument `validity` will be ignored for `defect` (or whatever listed in config `validityUntilSuperseded`), so 0 or 999 doesn't matter (DM-5500)